exe/server/main.go

```diff
@@ -32,19 +29,18 @@ var (
 )
 
 func main() {
```

Comment:
Improve description messages.
Eventually might worth switching to a more powerful CLI framework for the server? 
I wouldn't expect much flags to be available, but might provide nicier output of `--help` commands? 
Something to consider.

Reply:
Yea, I get sad whenever I see a CLI not using Cobra now 😢